nixos/modules/system/boot/networkd.nix

```
@@ -1777,6 +1824,10 @@ in
 
       systemd.services.systemd-networkd-wait-online = {
         wantedBy = [ "network-online.target" ];
+        serviceConfig.ExecStart = [
+          ""
+          "${config.systemd.package}/lib/systemd/systemd-networkd-wait-online ${concatStringsSep " " cfg.wait-online.extraArgs}"
```

maybe?
```suggestion
          "${config.systemd.package}/lib/systemd/systemd-networkd-wait-online ${utils.escapeSystemdExecArgs cfg.wait-online.extraArgs}"
```
also for the templated target. though this might be a case where we really *don't* want to escape, but that seems somewhat doubtful for either instance.